### ISSUE TITLE
"Player" tab on the web panel.

### ISSUE THREAD

**Issue by AdxmGG** [Minehut/Meta#287]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website

**Describe the feature**

- There will be three main tabs within the Players tab. **Online**, **Operators**, and **Banned**.
- The **Online** tab will show all online users, along with useful information such as their current world, their health or any other information that may serve useful. There will also be an **Action Menu**, which is a popup that can be opened by clicking **Actions** beside the users name. This will give options such as **Kick**, **Ban** and **Make Operator**.
- The **Operators** tab will show a list of users that are currently server operators. The date they were made an operator can be shown, as-well-as a button next to their name, **Remove Operator**, which will remove the player from the operator status.
- The **Banned** tab shows a list of users that are currently banned from the server. Their ban reason, date, and the user who banned them will be shown. There will also be a button, **Unban Player**, which will unban the player from the server.

**How the feature is useful**

This will benefit anybody who owns and runs a Minehut server. This is much easier and a lot clearer for users to control their server from the web panel.

**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by DaRealMinecraftexpert** [comment]
Would it show Offline Players too?

**Comment by TeamMH** [comment]
No, in my idea it would only show online players. In my opinion there would be way too many offline players for that to work efficiently.

**Comment by LukasmanMHdude** [comment]
I think that opped offline players would be a good idea though

**Comment by AdxmGG** [comment]
I think having offline players in general would be too much, however I think offline players should ahow in the operator  & banned tabs.

**Comment by Shlba** [comment]
I think this is a good idea for smaller servers, but for bigger ones with larger playerbases, it may crowd the control panel a bit.

**Comment by Blocksnmore** [comment]
Hence why it would be its own tab on the panel

**Comment by LukasmanMHdude** [comment]
I would hope this would be a new section on the panel

**Comment by AdxmGG** [comment]
I understand what you mean, but it could be paginated, and you could select how many players show up per page (default being 10).

**Comment by MiginsMC** [comment]
> I think this is a good idea for smaller servers, but for bigger ones with larger playerbases, it may crowd the control panel a bit.

then larger servers don't use it? I think this feature is mainly designed for those who aren't familiar with Minecraft commands and to make it easier for smaller servers to keep track of their players. 
As stated above it is suggested to be its own tab so I don't see how it would crowd the control panel at all.

**Comment by LukasmanMHdude** [comment]
eh I would use it until we get live console (hopefully with autofill?)

**Comment by Shlba** [comment]
> then larger servers don't use it? I think this feature is mainly designed for those who aren't familiar with Minecraft commands and to make it easier for smaller servers to keep track of their players.
As stated above it is suggested to be its own tab so I don't see how it would crowd the control panel at all.

No I mean it would be too crowded for a large server to keep track of

**Comment by MiginsMC** [comment]
> > then larger servers don't use it? I think this feature is mainly designed for those who aren't familiar with Minecraft commands and to make it easier for smaller servers to keep track of their players.
> > As stated above it is suggested to be its own tab so I don't see how it would crowd the control panel at all.
> 
> No I mean it would be too crowded for a large server to keep track of

then they don't use it?? I'm not sure what your point is

**Comment by AdxmGG** [comment]
I don't see how a bigger server would affect anything really. As I stated before, it could be paginated, sort users alphabetically, and there's no reason a bigger server should have any issues...

**Comment by drtshock** [comment]
what about the whitelist?

**Comment by Shlba** [comment]
> what about the whitelist?

:notlikethis:

**Comment by LukasmanMHdude** [comment]
yea that would be realy nice as well

:pepehatTip:

**Comment by AdxmGG** [comment]
> what about the whitelist?

I never thought about this, yeah **whitelist** could be a great implementation. It would show all players currently on the whitelist, along with the ability to remove them. Furthermore, you could add **Add to Whitelist** to the actions menu on the player tab.

**Comment by AdxmGG** [comment]
bump + added Whitelist tab to the main suggestjon

**Comment by mekb-turtle** [comment]
:-1: This can cause issues/conflicts with custom ban/whitelist Skripts and other plugins

**Comment by MiginsMC** [comment]
> 👎 This can cause issues/conflicts with custom ban/whitelist Skripts and other plugins

I don't see how this could do that at all if they have a custom ban/whitelist system (which tbh is a bad practice in general), they can just ignore this tab.

**Comment by AdxmGG** [comment]
Exactly what Mig said. Even though it *may* cause conflicts between custom made Skripts, it shouldn't stop the majority of users benefiting from this tab.

**Comment by LukasmanMHdude** [comment]
> Exactly what Mig said. Even though it _may_ cause conflicts between custom made Skripts, it shouldn't stop the majority of users benefiting from this tab.

majority of people use ab or essentials

**Comment by AdxmGG** [comment]
bump 😉

**Comment by Sniped** [comment]
> bump 😉

There's no need to make a comment solely for the purpose of bumping your issue. Admins get to all issues and read through them.

**Comment by AdxmGG** [comment]
> > bump 😉
> 
> There's no need to make a comment solely for the purpose of bumping your issue. Admins get to all issues and read through them.

bumped

**Comment by Prorickey** [comment]
> > Exactly what Mig said. Even though it _may_ cause conflicts between custom made Skripts, it shouldn't stop the majority of users benefiting from this tab.
> 
> majority of people use ab or essentials

I would say most servers have no plugins and are just quick smps for people and their friends.
